omnisend/includes/Internal/class-sync.php

```diff
@@ -20,31 +20,13 @@ class Sync {
 	 * @param $user_id
 	 * @return void
 	 */
-	public static function hook_user_register( $user_id ): void {
-		if ( \Omnisend_Core_Bootstrap::is_omnisend_woocommerce_plugin_connected() ) {
-			return; // do not sync if omni woo plugin is active.
-		}
-
-		$user = get_userdata( $user_id );
-		if ( $user ) {
-			self::sync_contact( $user );
-		}
-	}
-
-	/**
-	 * Listens for 'profile_update' hook https://developer.wordpress.org/reference/hooks/profile_update/
-	 *
-	 * @param $user_id
-	 * @return void
-	 */
-	public static function hook_profile_update( $user_id ): void {
-		if ( \Omnisend_Core_Bootstrap::is_omnisend_woocommerce_plugin_connected() ) {
-			return; // do not sync if omni woo plugin is active.
-		}
-
+	public static function hook_identify_user_by_id( $user_id ): void {
```

Comment:
Function prefix `hook_` has special meaning. It means that this function is registered and listens for WP hook without prefix. Please read function description.

Now `hook_identify_user_by_id` description is not correct. Please change to what function actually does like:
`sync_and_identify`